src/elements/date-input/date-input.ts

```diff
@@ -108,12 +114,24 @@ class SbbDateInputElement<T = Date> extends SbbFormAssociatedInputMixin(LitEleme
    * parsing of the string value.
    */
   private _valueCache?: [string, T | null];
+  private _dateAdapter: DateAdapter<T> = readConfig().datetime?.dateAdapter ?? defaultDateAdapter;
+  private _placeholderMutable = false;
+
+  /**
+   * Gets the associated datepicker, if any.
+   * The date-input and the datepicker are assumed to be in the same parent container.
+   */
+  private get _datepicker(): SbbDatepickerElement<T> | null {
```

Comment:
I think we once decided to not have private getters?
```suggestion
  private datepicker(): SbbDatepickerElement<T> | null {
```